src/clue/clue.sass

```
@@ -62,6 +62,20 @@
               background-image: url("../assets/icons/delete/delete-hover.svg")
           &:active
               background-image: url("../assets/icons/delete/delete-active.svg")
+        .button-icon.share
+          &.visibility.public
+            background-image: url("../assets/icons/share/not-share.svg")
```

```suggestion
            background-image: url("../assets/icons/share/share.svg")
```
The icons are reversed. If the visibility is `public` we should show the `share` icon by default and then the `not-share` icon on hover/active.